beacon-chain/core/peerdas/validator.go

```diff
@@ -95,17 +95,21 @@ func ValidatorsCustodyRequirement(state beaconState.ReadOnlyBeaconState, validat
 
 // DataColumnSidecars, given ConstructionPopulator and the cells/proofs associated with each blob in the
 // block, assembles sidecars which can be distributed to peers.
+// cellsPerBlob and proofsPerBlob are parallel slices where each index represents a blob.
```

Comment:
```suggestion
// cellsPerBlob and proofsPerBlob are parallel slices where each index represents a blob sidecar.
```

(There is no proof in a blob.)